### ISSUE TITLE
Unable to deploy Loadbalancer without description

### ISSUE THREAD

**Issue by blokje** [exoscale/exoscale-cloud-controller-manager#38]
Following the getting started guide. I get the following error while running the example yaml:
```yaml
kind: Service
apiVersion: v1
metadata:
  name: nginx
spec:
  selector:
    app: nginx
  type: LoadBalancer
  ports:
  - port: 80
```

```
$ k describe service nginx
Name:                     nginx
Namespace:                default
Labels:                   <none>
Annotations:              service.beta.kubernetes.io/exoscale-loadbalancer-service-instancepool-id: 1269a3f9-c088-4b0a-8b8a-f91bee2521be
Selector:                 app=nginx
Type:                     LoadBalancer
IP Family Policy:         SingleStack
IP Families:              IPv4
IP:                       10.106.8.90
IPs:                      10.106.8.90
Port:                     <unset>  80/TCP
TargetPort:               80/TCP
NodePort:                 <unset>  32140/TCP
Endpoints:                192.168.33.71:80,192.168.33.72:80
Session Affinity:         None
External Traffic Policy:  Cluster
Events:
  Type     Reason                  Age                From                Message
  ----     ------                  ----               ----                -------
  Normal   EnsuringLoadBalancer    25s (x4 over 51s)  service-controller  Ensuring load balancer
  Warning  SyncLoadBalancerFailed  25s (x4 over 50s)  service-controller  Error syncing load balancer: failed to ensure load balancer: invalid request: the description field is incorrect
```

After adding the following annotation the loadbalancer gets deployed

```yaml
kind: Service
apiVersion: v1
metadata:
  name: nginx
  annotations:
    service.beta.kubernetes.io/exoscale-loadbalancer-description: "Test"
spec:
  selector:
    app: nginx
  type: LoadBalancer
  ports:
  - port: 80
```

```
$ k describe service nginx
Name:                     nginx
Namespace:                default
Labels:                   <none>
Annotations:              service.beta.kubernetes.io/exoscale-loadbalancer-description: Test
                          service.beta.kubernetes.io/exoscale-loadbalancer-id: 01ec23f8-b37c-43fa-98df-b3889a40b8dc
                          service.beta.kubernetes.io/exoscale-loadbalancer-service-instancepool-id: 1269a3f9-c088-4b0a-8b8a-f91bee2521be
Selector:                 app=nginx
Type:                     LoadBalancer
IP Family Policy:         SingleStack
IP Families:              IPv4
IP:                       10.106.8.90
IPs:                      10.106.8.90
LoadBalancer Ingress:     194.182.171.XXX
Port:                     <unset>  80/TCP
TargetPort:               80/TCP
NodePort:                 <unset>  32140/TCP
Endpoints:                192.168.33.71:80,192.168.33.72:80
Session Affinity:         None
External Traffic Policy:  Cluster
Events:
  Type     Reason                  Age                 From                Message
  ----     ------                  ----                ----                -------
  Warning  SyncLoadBalancerFailed  61s (x5 over 2m6s)  service-controller  Error syncing load balancer: failed to ensure load balancer: invalid request: the description field is incorrect
  Normal   EnsuringLoadBalancer    36s (x6 over 2m7s)  service-controller  Ensuring load balancer
  Normal   EnsuredLoadBalancer     21s                 service-controller  Ensured load balancer
```

```
┼───────────────┼──────────────────────────────────────────────────────────────────┼
│  SKS CLUSTER  │                                                                  │
┼───────────────┼──────────────────────────────────────────────────────────────────┼
│ ID            │ 2bb4e2f7-c617-4e6f-bc84-aaaaaaaaaaaa                             │
│ Name          │ kubernetes01                                                     │
│ Description   │                                                                  │
│ Zone          │ de-fra-1                                                         │
│ Creation Date │ 2021-11-01 14:59:46 +0000 UTC                                    │
│ Endpoint      │ https://2bb4e2f7-c617-4e6f-bc84-aaaaaaaaaaaa.sks-de-fra-1.exo.io │
│ Version       │ 1.22.2                                                           │
│ Service Level │ pro                                                              │
│ CNI           │ calico                                                           │
│ Add-Ons       │ exoscale-cloud-controller                                        │
│               │ metrics-server                                                   │
│ State         │ running                                                          │
│ Labels        │ n/a                                                              │
│ Nodepools     │ 6de49cbe-47bc-41c6-a205-9bd365069cbc | nodepool-standard-medium  │
┼───────────────┼──────────────────────────────────────────────────────────────────┼
$ exo compute sks nodepool  show kubernetes01 nodepool-standard-medium 
┼──────────────────────┼──────────────────────────────────────┼
│     SKS NODEPOOL     │                                      │
┼──────────────────────┼──────────────────────────────────────┼
│ ID                   │ 6de49cbe-47bc-41c6-a205-9bd365069cbc │
│ Name                 │ nodepool-standard-medium             │
│ Description          │                                      │
│ Creation Date        │ 2021-11-01 15:10:15 +0000 UTC        │
│ Instance Pool ID     │ 1269a3f9-c088-4b0a-8b8a-f91bee2521be │
│ Instance Prefix      │ pool                                 │
│ Instance Type        │ medium                               │
│ Template             │ SKS Node (kubelet 1.22.2)            │
│ Disk Size            │ 50                                   │
│ Anti Affinity Groups │ kubecluster                          │
│ Security Groups      │ sks-kubernetes01                     │
│ Private Networks     │ n/a                                  │
│ Version              │ 1.22.2                               │
│ Size                 │ 2                                    │
│ State                │ running                              │
│ Taints               │ n/a                                  │
│ Labels               │ n/a                                  │
┼──────────────────────┼──────────────────────────────────────┼
```

**Comment by falzm** [comment]
Hello @blokje, thank you for reporting this issue. This is a known bug in the Exoscale API that we are currently working on fixing, in the meantime as a workaround indeed you must set a value for the `service.beta.kubernetes.io/exoscale-loadbalancer-description` annotation.

We apologize for the inconvenience.

**Comment by blokje** [comment]
Hi @falzm, thanks for the quick reply. Feel free to close the issue now or once you've managed to fix the bug in the Exoscale API. Glad I'm at least doing nothing wrong.

PS. Really starting to love your platform, it feels to responsive and fast 👍

**Comment by falzm** [comment]
Hi @blokje, sorry I forgot to get back to you – this problem should be fixed starting from version 0.8.0.
